README.md

```diff
@@ -132,7 +132,7 @@ You can deploy this project using:
 
```

Comment:
Great job @Kasaija-Kenneth 🤝

- [ ] [Optional] Could you fix the broken MIT license link to make your readme file more professional?
- [ ] [Optional] You may also remove the live demo link since this project is not yet deployed.

Reply:
Noted and thanks @1GORDON